tcms/issuetracker/types.py

```diff
@@ -344,3 +344,79 @@ def report_issue_from_testexecution(self, execution, user):
         )
 
         return new_url
+
+
+class AzureDevops(IssueTrackerType):
```

Comment:
Move to azure_boards.py, rename to AzureBoards, see original description. There is no need to have the integration classes added to this file anymore.